src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/FailOverConstants.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using System;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Constants
+{
+    internal class FailOverConstants
+    {
+        // Timeouts to retry requests to config stores and their replicas after failure.
+        public static readonly TimeSpan MinBackoffInterval = TimeSpan.FromSeconds(30);
```

Comment:
"Duration" feels more accurate than "Interval" here. How about `MinBackoffDuration`?

Reply:
I am curious to know why `Duration` is more accurate?

Reply:
"Interval" refers to the period between _repeated_ events, for example, we revalidate the cache once its expiration passes, and we will put a new expiration on it after validation. Here we just put an endpoint on hold for some time, and we may or may not come back for it when the period ends. If we do come back, we don't put it on hold again regardless (i.e., the backoff doesn't always repeat).

Reply:
Thank you for the explanation. Updated!